src/tools/rbd/rbd.cc

```diff
@@ -5,6 +5,10 @@
 
 int main(int argc, const char **argv)
 {
+  #ifdef _WIN32
+  setlocale(LC_ALL, ".UTF8");
```

Comment:
I was looking at this note: https://learn.microsoft.com/en-us/windows/apps/design/globalizing/use-utf8-code-page#code-page-conversion

> CP_ACP equates to CP_UTF8 only if running on Windows Version 1903 (May 2019 Update) or above and the ActiveCodePage property described above is set to UTF-8. Otherwise, it honors the legacy system code page. We recommend using CP_UTF8 explicitly.

But I think it was actually referring to the ``MultiByteToWideChar`` and ``WideCharToMultiByte`` functions. I'll go ahead and drop the ``setlocale(LC_ALL, ".UTF8");`` calls.

``SetConsoleOutputCP(CP_UTF8);`` is still needed though, otherwise it seems to use ANSI console output even with the code page set to utf-8.